### DISCUSSION TITLE
Extended Support for Istio 1.18.5

### DISCUSSION THREAD

**Discussion by altdelnow** [istio/istio#48741] [General]
Hi Community,

We are long time user of Istio in our prod environment and recently upgraded to 1.18.5. But as this version has gone out of support on 31st Dec 2023, we are suppose to upgrading to 1.19.x which again go EoL on 31st March 2024. These frequent upgrades has an impact on prod environment. Is it possible for Istio to provided extended support for atleast 6 months on n-2 versions which will help users to run the installed version for longer duration without frequent upgrade.

**Comment by howardjohn** [comment]
Hi. Istio quite regularly discusses changes to our release and support schedules, including the suggestion here and more. Currently there are not any plans to make changes, but this topic comes up frequently so it's plausible it could change in the future.

That being said, an Istio vendor (https://istio.io/latest/about/ecosystem/) may be suitable for you as these provide actual support (paid) and tend to offer extended support windows.

**Reply by altdelnow** [reply]
Thanks @howardjohn for your reply.

**Reply by altdelnow** [reply]
@howardjohn Let me thank you and the community for extending the support cycle for Istio release from 3 month to 8 months now which has greatly helped us in cutting down frequent upgrades and disruptions. This has also helped us in aligning EKS upgrades with Istio upgrade as both now have similar support cycle.
